assets/dev/js/frontend/handlers/base-nested-tabs.js

```diff
@@ -154,10 +154,20 @@ export default class BaseNestedTabs extends Base {
 	activateTab( tabIndex ) {
```

Comment:
I realised that we missed something and that we'll need to double check the mindmap.

This function `activateTab()` is used in various other places as well and we'll need to implement the same change everywhere, such as in the the general repeater base, the mega menu, and possibly in other widgets as well.